sql/core/src/test/scala/org/apache/spark/status/api/v1/sql/SqlResourceSuite.scala

```diff
@@ -204,4 +203,25 @@ class SqlResourceSuite extends SparkFunSuite with PrivateMethodTester {
     assert(wholeStageCodegenId == Some(1))
   }
 
+  test("Status of execution w/ error and w/o jobs shall be FAILED not COMPLETED") {
```

Comment:
Could you prepend `SPARK-44334: ` to the test name?

Reply:
addressed, and I add a new test case with a DDL.